src/main/java/org/openpnp/machine/reference/vision/wizards/ReferenceFiducialLocatorConfigurationWizard.java

```diff
@@ -0,0 +1,233 @@
+package org.openpnp.machine.reference.vision.wizards;
```

Comment:
The labels in the wizard make it too wide for a common laptop screen. Screenshot:
![screen shot 2017-02-06 at 7 23 38 pm](https://cloud.githubusercontent.com/assets/1182323/22673843/3871b25e-eca2-11e6-8943-5fdbde5f9e30.png)

You can just add some newlines to the labels to get the total width down. Shoot for the program to look good on a 1440x900 screen.